### PR TITLE
Implement Basic Authorization feature

### PR DESCRIPTION
**Pull request by Aashruti20**
These code changes would implement basic authorisation. UI includes Basic option in the Auth dropdown including username and password input fields. Once the user enters userName and password the headers will getting updated.

### CODE REVIEW

File: src/components/AuthFields.jsx [hunk 1]
```diff
@@ -3,22 +3,55 @@ import PropTypes from "prop-types";
 import PureTextInput from "./PureTextInput";
 
 export const AuthFields = ({ authType, onHeaderChange }) => {
+	const [userName, setUserName] = useState("");
```
Comment (VaishnaviKillekar) [inline src/components/AuthFields.jsx]: Let's rename this to `username` instead and update the method to `setUsername`. Update the same in all usages

**Review by VaishnaviKillekar — CHANGES_REQUESTED** [review]
Please include this feature along with the bearer token in the readme

**Review by omkarajagunde — APPROVED** [review]
LGTM...

File: src/components/AuthFields.jsx (hunk 1 above)
Comment (Aashruti20) [reply src/components/AuthFields.jsx]: Updated

**Review by VaishnaviKillekar — APPROVED** [review]
Looks good

**Review by ashrithshetty — APPROVED** [review]

**Pull request merged**